prow/README.md

```diff
@@ -7,31 +7,132 @@ for CI automation.
 - Prow runs in [the tektoncd GCP project](../gcp.md)
 - [Ingress is configured to `prow.tekton.dev`](#ingress)
 - Prow results are displayed via [gubernator](../gubernator/README.md)
+- [Instructions for creating the Prow cluster](#creating-the-prow-cluster)
 - [Instructions for updating Prow](#updating-prow-itself) and [Prow's Tekton Pipelines instance](#tekton-pipelines-with-prow)
 - [Instructions for updating Prow configuration](#updating-prow-configuration)
 
 _[Prow docs](https://github.com/kubernetes/test-infra/tree/master/prow)._
 _[See the community docs](../CONTRIBUTING.md#pull-request-process) for more on
 Prow and the PR process._
 
-### Secrets
+## Creating the Prow cluster
 
-- Prow uses the service account
-  `prow-account@tekton-releases.iam.gserviceaccount.com`
-- Secrets for this account are configured in [Prow's config.yaml](config.yaml) via
-  `gcs_credentials_secret: "test-account"`
+If you need to re-create the Prow cluster (which includes [the boskos](../boskos/README.md)
+running inside), you will need to:
 
-## Ingress
+1. [Create a new cluster](#creating-the-cluster)
+2. [Create the necessary secrets](#creating-the-secrets)
+3. [Apply the new Prow and Boskos](#start-it)
+4. [Setup ingress](#ingress)
+4. [Update GitHub webhook(s)](#update-github-webhook)
 
-- Ingress for prow is configured using
-  [cert-manager](https://github.com/jetstack/cert-manager/).
-- `cert-manager` was installed via `Helm` using this
+### Creating the cluster
+
+To create a cluster of the right size, using [the same GCP project](../gcp.md):
+
+```bash
+export PROJECT_ID=tekton-releases
+export CLUSTER_NAME=tekton-plumbing
+
+gcloud container clusters create $CLUSTER_NAME \
+ --scopes=cloud-platform \
+ --enable-basic-auth \
+ --issue-client-certificate \
+ --project=$PROJECT_ID \
+ --region=us-central1-a \
+ --machine-type=n1-standard-4 \
+ --image-type=cos \
+ --num-nodes=8 \
+ --cluster-version=latest
+```
+
+### Creating the secrets
+
+In order to operate, Prow needs the following secrets, which are referred
+to by the following names in out config:
+
+- `GCP` secret: `test-account` is a token for the service account
+  `prow-account@tekton-releases.iam.gserviceaccount.com`. This account can
+   interact with GCP resources such as uploading Prow results to GCS
+   (which is done directly from the containers started by Prow, configured in [config.yaml](config.yaml)) and
+   [interacting with boskos clusters](../boskos/README.md).
+- `Github` secrets: `hmac-token` for authtenticating GitHub and `oauth-token` which is a
```

Comment:
`authtenticating` -> `authenticating`

Reply:
wow i can barely see the difference XD :eyeglasses: